app/controllers/keycloak.ts

```diff
@@ -1,5 +1,8 @@
-import { RoleMappingPayload } from 'keycloak-admin/lib/defs/roleRepresentation';
+import ClientRepresentation from 'keycloak-admin/lib/defs/clientRepresentation';
```

Comment:
This lib is archived now, I wonder if it makes sense while we have time to try out the new one. Probably a different ticket though